zoomus/components/poll.py

```diff
@@ -0,0 +1,63 @@
+"""Zoom.us REST API Python Client -- Recording component"""
+from zoomus import util
+from zoomus.components import base
+
+
+class PollComponentV2(base.BaseComponent):
+    def __init__(self, *args, **kwargs):
+        util.require_keys(kwargs, "type")
+        self.type = kwargs.get('type')
+        super().__init__(*args, **kwargs)
+
+
+    def list(self, **kwargs):
+        util.require_keys(kwargs, "id")
+        return self.get_request(
+            f"/{self.type}/{kwargs.get('id')}/polls"
```

Comment:
Looks like in some cases you are using `f"...."` and in others `"...".format()`. Since the rest of the library uses the `"...".format()` syntax, can you please switch it to that for consistency's sake?

Reply:
Sure, I have changed the ` f"...." `and in this `"...".format()`